gradle/scripts/ember.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: "com.moowork.node"
 
 node {
   // Version of node to use.
-  version = '7.8.0'
+  version = '7.10.1'
```

Comment:
It builds for me from a clean checkout. I believe a gralde clean will fix the issue for them.